migrations/1_create_users_projects_files.js

```diff
@@ -8,6 +8,10 @@ exports.up = function (knex) {
       t.increments('id');
       t.integer('user_id').notNullable().references('id').inTable('users');
       t.text('title').notNullable();
+      t.text('tags');
```

Comment:
when we do this in pg it shouldn't be so bad. http://www.postgresql.org/docs/9.1/static/arrays.html